src/mesh/index_derivs.cxx

```diff
@@ -2305,87 +2191,15 @@ const Field3D Mesh::indexVDDY(const Field3D &v, const Field3D &f, CELL_LOC outlo
           fval.c = f[i];
           fval.p = f.yup()[i.yp()];
 
-          if (diffloc != CELL_DEFAULT) {
-            // Non-centred stencil
-            if ((vloc == CELL_CENTRE) && (diffloc == CELL_YLOW)) {
-              // Producing a stencil centred around a lower Y value
-              vval.pp = vval.p;
-              vval.p = vval.c;
-            } else if (vloc == CELL_YLOW) {
-              // Stencil centred around a cell centre
-              vval.mm = vval.m;
-              vval.m = vval.c;
-            }
-            // Shifted in one direction -> shift in another
-            // Could produce warning
-          }
-          result[i] = func(vval, fval);
-        }
-      }
-    } else if (vUseUpDown) {
```

Comment:
What happened to this branch?

Reply:
Sorry, should have put this in a separate commit with an explanation: it's not correct to mix fields with yup/ydown and fields that are shifted to field-aligned, because the physical locations of their grid points are different. So I just got rid of them rather than updating the diffloc, etc. in those branches, but I should have been not quite so lazy.